control/controlclient/map.go

```diff
@@ -35,7 +35,7 @@ type mapSession struct {
 	machinePubKey          key.MachinePublic
 	keepSharerAndUserSplit bool // see Options.KeepSharerAndUserSplit
 
-	// Fields storing state over the the coards of multiple MapResponses.
+	// Fields storing state over the coords of multiple MapResponses.
```

Comment:
I'm fairly confident about `the`. I couldn't figure out the other word at all, so this is mostly to hang a question: "what is this?"

Reply:
Over the course of